server/controllers/cookieController.js

```diff
@@ -0,0 +1,14 @@
+const db = require('../models/quizModels');
```

Comment:
I don't think `db` is being used. Do we still need it here?

Reply:
Agreed, removed

Reply:
Looks good.